src/projections_site/templates/crs.html.j2

```diff
@@ -17,14 +17,21 @@
 
     <h3>Area of use</h3>
     <p>{{ crs.area_of_use.name }}</p>
-
-    {# TODO:
-    <p>Geodetic CRS: {{ util.get_geodetic_crs(crs) }}</p>
-    <p>Datum: {{ util.get_datum_name(crs) }}</p>
     <p>Data Source: EPSG  {{ util.get_epsg_version_string() }}</p>
```

Comment:
Thinking out loud: The datasource may not always be the EPSG database, for example I think IAU codes are from another source. I'll open an issue to revisit this question :)